Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
Do we have any precedent set forth in other controls? So far there are several name candidates: AccentColor, HighlightColor, AccentBrush, ButtonBackground, HighlightBrush, HamburgerButtonBackground.